charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response - GetAssetDetails",
+  "type": "object",
+  "properties": {
+    "name": {
+      "$ref": "#/definitions/assetName"
```

Comment:
What's the difference between this field and asset.assetMetadata.metadata.name?

Reply:
removed the `name` field as it is covered by the `resourceMetadata.name` field. Thanks.